webapp/src/components/createCategory/createCategory.tsx

```diff
@@ -71,6 +72,7 @@ const CreateCategory = (props: Props): JSX.Element => {
             } as Category
 
             await mutator.createCategory(category)
```

Comment:
To get the new category ID you need to make just 2 changes, you need here a `newCategory =  await mutator.createCategory(category)` and add in the `mutator.createCategory` function definition a return the data, probably with some code like:

```ts
                  const res = await octoClient.createSidebarCategory(category)
                  const jsonres = await res.json()
                  const newCategory = jsonres as Category
                  return newCategory

```

The API is already returning it.

Reply:
Ok I see. I just assume that server doesn't return it. I just apply your review. 

Thank you for help